src/nvim/eval.c

```diff
@@ -6581,6 +6581,7 @@ static struct fst {
   {"printf",          2, 19, f_printf},
   {"pumvisible",      0, 0, f_pumvisible},
   {"pyeval",          1, 1, f_pyeval},
+  {"py3eval",         1, 1, f_py3eval},
```

Comment:
> "E117: Unknown function: pyeval"

Function lookup is done with binary search, so `py3eval` must come before `pyeval`